org.knime.knip.io/src/org/knime/knip/io/nodes/imgreader2/readfrominput/ReadImgTableFunction.java

```diff
@@ -76,7 +76,7 @@ public ReadImgTableFunction(ExecutionContext exec, int numberOfFiles, SettingsMo
 					|| url.getProtocol().equalsIgnoreCase("HTTPS")) {
```

Comment:
do we need this explicit check at all anymore? the else condition should handle all cases, right?